packages/cli/src/commands/ui/create/atomic.ts

```diff
@@ -87,6 +93,8 @@ export default class Atomic extends Command {
       username,
     ];
 
+    this.args.pageId && cliArgs.push('--page-id', this.args.pageId);
```

Comment:
Yeah I know, I know, this is common in React, but
![image](https://user-images.githubusercontent.com/12366410/160751608-f6ecfcc5-3899-46c7-8c83-9cf9b392eac0.png)
```suggestion
    if (this.args.pageId) {
      cliArgs.push('--page-id', this.args.pageId);
    }
```